sf_stats.Api/sf_stats.Domain/Entities/Division.cs

```diff
@@ -11,5 +7,7 @@ public class Division : DbEntity
         public int Id { get; set; }
         public string Code { get; set; }
         public string DisplayName { get; set; }
+
+        public List<Game> Games { get; set; }
```

Comment:
should this be a virtual property?

```suggestion
        public virtual List<Game> Games { get; set; }
```